### PR TITLE
Update Adapter Change Process to use main branch

### PR DESCRIPTION
**Pull request by kbenzie**
Depends on #1159

### CODE REVIEW

**Review by omarahmed1111 — APPROVED** [review]

**Comment by omarahmed1111** [comment]
I think we need to update the nightly CI also [here](https://github.com/oneapi-src/unified-runtime/blob/9e3bb2f902181c1ea62d8e58315c61174e06a626/.github/workflows/nightly.yml#L22)

**Comment by kbenzie** [comment]
> I think we need to update the nightly CI also [here](https://github.com/oneapi-src/unified-runtime/blob/9e3bb2f902181c1ea62d8e58315c61174e06a626/.github/workflows/nightly.yml#L22)

Good shout! Thanks.

File: .github/workflows/nightly.yml [hunk 1]
```diff
@@ -17,9 +17,6 @@ jobs:
 
     steps:
     - uses: actions/checkout@v3
-      # with-ref part to be removed after merging 'adapters' branch with 'main'
```
Comment (fabiomestre) [inline .github/workflows/nightly.yml:20]: There is something similar in the e2e nightly which probably needs to be removed as well

Comment (kbenzie) [reply .github/workflows/nightly.yml:20]: I'll take a look

Comment (kbenzie) [reply .github/workflows/nightly.yml:20]: Removed.

**Review by martygrant — APPROVED** [review]

**Pull request merged**